### PR TITLE
feat: Fixed confetti controller dispose issue

### PR DESCRIPTION
**Pull request by M4dhav**
Made required fixes as per #104

### CODE REVIEW

**Comment by M4dhav** [comment]
The reason this was happening was that Dispose was being called twice, once within the ConfettiAtom and then on the Individual pages themselves. Since we were passing the same controller and not using Copy Constructors, it was already disposed before the Page Destructor reached it.

**Review by guyluz11 — CHANGES_REQUESTED** [review]

**Comment by guyluz11** [comment]
I have another easy one for you 
https://github.com/guyluz11/infinite_horizons/issues/101

File: lib/presentation/atoms/confetti_atom.dart [hunk 1]
```diff
@@ -13,12 +13,6 @@ class ConfettiAtom extends StatefulWidget {
 }
 
 class _ConfettiAtomState extends State<ConfettiAtom> {
-  @override
-  void dispose() {
-    widget.controllerCenter.dispose();
-    super.dispose();
-  }
```
[the remark this answers is not in the document]
Comment (M4dhav) [reply lib/presentation/atoms/confetti_atom.dart:20]: Very well

File: lib/presentation/atoms/confetti_atom.dart [hunk 2]
```diff
@@ -18,7 +18,6 @@ class _ConfettiAtomState extends State<ConfettiAtom> {
     widget.controllerCenter.dispose();
     super.dispose();
   }
-
```
Comment (guyluz11) [inline lib/presentation/atoms/confetti_atom.dart:21]: Redundant change and extra space here is better.

**Review by guyluz11 — APPROVED** [review]

**Comment by guyluz11** [comment]
Also commit with the name "Requested changes have been made" does not explain what got done in the commit.

**Comment by guyluz11** [comment]
Please comment on the issue so that I can close it with you assign to it

**Pull request merged**